persistence/rocksdb/src/main/java/org/infinispan/persistence/rocksdb/configuration/RocksDBExpirationConfiguration.java

```diff
@@ -47,7 +51,12 @@ public String expiredLocation() {
       return expiredLocation.get();
    }
 
+   /**
+    * @deprecated Since 10.1, there is no more queue in {@link org.infinispan.persistence.rocksdb.RocksDBStore}
+    */
+   @Deprecated
    int expiryQueueSize() {
+      log.warn("The expiry queueSize attribute is deprecated and will be removed");
```

Comment:
@diegolovison the warning should be logged when setting the queue size, not when reading it.

This is what I had in mind: https://github.com/infinispan/infinispan/commit/4b40bf35dfbd892fb993b9096a30164ec0a4af39

Reply:
@danberindei done. thanks for the explanation.